pyanaconda/payload/dnfpayload.py

```diff
@@ -557,6 +563,8 @@ def _configure(self):
         dnf_logger = get_dnf_logger()
         dnf_logger.setLevel(dnf.logging.DDEBUG)
 
+        log.debug("Dnf configuration:\n%s", conf.dump())
```

Comment:
Please use `format` to add `core.dump()` to the text.

Reply:
Fixed.